_data/table.csv

```diff
@@ -1,74 +1,74 @@
-0.2,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.2/6001240.zarr,271,275,236,2,1,XYZCT,,labels (0),CC BY 4.0,idr0062,,2021-03-09
-0.2,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.2/6001247.zarr,253,210,257,2,1,XYZCT,,labels (0),CC BY 4.0,idr0062,,2020-12-01
-0.3,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/9836842.zarr,1920,1920,,4,,XYC,,,CC BY 4.0,idr0077,,2021-12-16
-0.3,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/idr0079A/9836998.zarr,1584,788,142,2,,XYZC ,,labels (0),CC BY 4.0,idr0079,,2021-12-16
-0.3,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/idr0052A/5514375.zarr,256,256,31,3,40,XYZCT,,"labels (cells, chromosomes)",CC BY 4.0,idr0052,,2021-12-16
-0.3,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/idr0095B/11511419.zarr,2048,2048,,3,,XYC,,labels (0),CC BY 4.0,idr0095,,2021-12-16
-0.3,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/idr0095B/11511420.zarr,2048,2048,,3,,XYC,,labels (0),CC BY 4.0,idr0095,,2021-12-16
-0.3,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/idr0095B/11511421.zarr,2048,2048,,3,,XYC,,labels (0),CC BY 4.0,idr0095,,2021-12-16
-0.3,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/idr0095B/11511422.zarr,2048,2048,,3,,XYC,,labels (0),CC BY 4.0,idr0095,,2021-12-16
-0.3,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/idr0095B/11511423.zarr,2048,2048,,3,,XYC,,labels (0),CC BY 4.0,idr0095,,2021-12-16
-0.3,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/idr0095B/11511424.zarr,2048,2048,,3,,XYC,,labels (0),CC BY 4.0,idr0095,,2021-12-16
-0.3,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/idr0094A/7751.zarr,1080,1080,,,,XY,96,labels,CC0 1.0,idr0094,,2021-12-16
-0.3,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/idr0109A/12922361.zarr,2560,2160,,,721,XYT,,,CC BY 4.0,idr0109,,2021-12-16
-0.3,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/idr0075A/9528933.zarr,512,512,91,,,XYZ,,,CC BY 4.0,idr0075,,2021-12-16
-0.3,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/idr0051A/4007817.zarr,333,333,201,,79,XYZT,,,CC BY 4.0,idr0051,,2021-12-16
-0.3,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/idr0040A/3491626.zarr,2048,2048,,5,20,XYCT,,,CC BY 4.0,idr0040,,2021-12-16
-0.4,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0073A/9798462.zarr/,21115,16433,1,3,1,XYZCT,,,CC BY 4.0,idr0073,,2022-05-05
-0.4,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0072B/9512.zarr,1345,1017,1,2,1,XYC,72,,CC BY 4.0,idr0072,,2022-05-10
-0.4,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0083A/9822152.zarr,144384,93184,1,1,1,XYZCT,,,CC BY 4.0,idr0083,,2022-05-11
-0.4,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0062A/6001247.zarr,253,210,257,2,,XYZC,,labels (0),CC BY 4.0,idr0062,,2022-05-11
-0.4,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0044A/4007801.zarr,2169,2048,988,2,532,XYZCT,,,CC BY 4.0,idr0044,,2022-05-17
+OME-NGFF version,EMBL-EBI bucket (current),SizeX,SizeY,SizeZ,SizeC,SizeT,Axes,Wells,Fields,Keywords,License,Study,DOI,Date added,Thumb ID

```

Comment:
Note that unlike wells, this will only work if the number of fields of view per well is homogenous. This is the case in most examples but might lead to truncation in other cases. An alternative would be to capture the total number of multiscale groups/images?

Reply:
I think that Fields is still useful even if it's not enforced to be the same for every Well.
Total number of Images takes more time to calculate and then you have to divide by the Wells to get Fields per Well.
With it being in the table, I can sort the table to find a Plate with lots of Fields per Well.